### ISSUE TITLE
Let Rector\Rector\Dynamic\ArgumentRector add argument

### ISSUE THREAD

**Issue by TomasVotruba** [rectorphp/rector#398]
Based on #353 

```yaml
    # @todo, use this and add new argument to a method
    Rector\Rector\Dynamic\ArgumentRector:
        $argumentByMethod:
             # Add state argument to the getLastPayment($state), where $state is target last payment state. getLastPayment(PaymentInterface::STATE_NEW)
            -
                type: 'add'
                class: 'Sylius\Component\Core\Model\OrderInterface'
                method: 'getLastPayment'
                position: 0 # as append | default value is 0, config argument need not be listed
                typehint: false # default value is false, config argument need not be listed
                property: '$state'
                default_value: PaymentInterface::STATE_NEW
```

**Comment by mssimi** [comment]
I wanted to do it, but I noticed type: 'changed' already does.

**Comment by TomasVotruba** [comment]
Allright, it was an easy pick :)

**Comment by TomasVotruba** [comment]
It was created due to this commit: https://github.com/rectorphp/rector/pull/353/commits/4345eadcb8ef950b724cd2573ac54ba551425666#diff-835696a01b6e3086012225ecb66ae9bbL75

**So to close this issue, `sylius10.yml` set needs to be updated with those 4 removed rules**. 
Could you do that after #451, so we don't have to re-investigate this all over again?

**Comment by mssimi** [comment]
I think new issue should be opened as it is not topic of this issue.

**Comment by TomasVotruba** [comment]
It would loose the context like this one and that might lead to unncessary re-investigation like you did.
